src/enum_transformer.ts

```diff
@@ -102,6 +103,9 @@ export function enumTransformer(typeChecker: ts.TypeChecker, diagnostics: ts.Dia
       // this is left to fix for another day.
       if (isInNamespace(node)) return ts.visitEachChild(node, visitor, context);
 
+      // Ambient enums should not be materialized.
```

Comment:
Can you put in a ", because [reason]" here?  It's not immediately obvious to me.

Reply:
Done - it's because TS doesn't produce values for an ambient enum either. Previously we'd accidentally generate values on the Closure side, which doesn't match vanilla TS then.